hack/make.sh

```diff
@@ -107,6 +107,15 @@ if \
 	DOCKER_BUILDTAGS+=' btrfs_noversion'
 fi
 
+# test whether "libdevmapper.h" is new enough to support deferred remove
+# functionality.
+if \
+	command -v gcc &> /dev/null \
+	&& ! ( echo -e  '#include <libdevmapper.h>\nint main() { dm_task_deferred_remove(NULL); }'| gcc -ldevmapper -xc - &> /dev/null ) \
+; then
+       DOCKER_BUILDTAGS+=' libdm_no_deferred_remove'
```

Comment:
Oh. I thought you were commenting on the build tag itself.
On Jun 5, 2015 7:55 PM, "Tianon Gravi" <notifications@github.com> wrote:

> In hack/make.sh
> <https://github.com/docker/docker/pull/12190#discussion_r31860689>:
>
> > @@ -107,6 +107,15 @@ if \
> >  	DOCKER_BUILDTAGS+=' btrfs_noversion'
> >  fi
> >
> > +# test whether "libdevmapper.h" is new enough to support deferred remove
> > +# functionality.
> > +if \
> > +	command -v gcc &> /dev/null \
> > +	&& ! ( echo -e  '#include <libdevmapper.h>\nint main() { dm_task_deferred_remove(NULL); }'| gcc -ldevmapper -xc - &> /dev/null ) \
> > +; then
> > +       DOCKER_BUILDTAGS+=' libdm_no_deferred_remove'
>
> The line itself is indented with eight spaces instead of one tab.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/12190/files#r31860689>.
>